ui-ngx/src/app/modules/home/pages/rulechain/rulechain-page.component.html

```diff
@@ -111,15 +112,21 @@
                               [ruleChainId]="ruleChain.id?.id"
                               [ruleChainType]="ruleChainType"
                               [isEdit]="true"
-                              [isReadOnly]="false">
+                              [isReadOnly]="false"
+                              [debugEventBody]="debugEventBody">
                 </tb-rule-node>
               </mat-tab>
               <mat-tab *ngIf="editingRuleNode.ruleNodeId" label="{{ 'rulenode.events' | translate }}" #eventsTab="matTab">
                 <tb-event-table [debugEventTypes]="[debugEventTypes.DEBUG_RULE_NODE]"
                                 [defaultEventType]="debugEventTypes.DEBUG_RULE_NODE"
                                 [active]="eventsTab.isActive"
                                 [tenantId]="ruleChain.tenantId.id"
-                                [entityId]="editingRuleNode.ruleNodeId"></tb-event-table>
+                                [entityId]="editingRuleNode.ruleNodeId"
+                                [isRuleNodeDebugModeEnabled]="editingRuleNode?.debugMode"
+                                [editingRuleNodeHasScript]="editingRuleNodeHasScript"
+                                [rulenodeClazz]="editingRuleNode.component.clazz"
```

Comment:
It is better to simplify the two changes before passing the name of the action, and if there is one, then show it.